merkle-tree/config/default.js

```diff
@@ -3,33 +3,23 @@
 @author iAmMichaelConnor
 @desc constants used by a nubmer of other modules
 */
-
-let nodeHashLength;
-let zero;
-
-if (process.env.HASH_TYPE === 'mimc') {
-  nodeHashLength = 32;
-  if (process.env.CURVE === 'BLS12_377') {
-    zero = 0;
-  } else {
-    zero = '0x0000000000000000000000000000000000000000000000000000000000000000';
```

Comment:
@MirandaWood - why was zero a decimal for altbn254, but hex for bls12-377?
I just want to make sure Duncan's neat refactoring here is correct.

Reply:
IIRC it's just the way the MiMC hash functions are written. The whole if statement assigns zero = 0 for MiMC hashing for all curves apart from altbn, because that has an older function which expects a (32bit) hex. Otherwise its a 27 bit hex for all curves under SHA. That may not be true anymore, though, I haven't had a proper look at the node/solidity MiMC in a while!